scripts/generators/asciidoc_fields.py

```diff
@@ -1,15 +1,54 @@
-from os.path import join
+from functools import wraps
+import os.path as path
```

Comment:
Thanks for this small adjustment. The out of the blue `join()` calls always looked strange to me 😂